x-pack/legacy/plugins/index_management/public/app/services/breadcrumbs.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import { BASE_PATH } from '../../../common/constants';
+
+class BreadcrumbService {
+  private chrome: any;
```

Comment:
We should have types for this, from here: `src/core/public/chrome/chrome_service.tsx`. I'm guessing the `ChromeStart` interface is the one we want.